gamestonk_terminal/forex/fx_view.py

```diff
@@ -0,0 +1,544 @@
+        response = client.request(request)
+        print(response)
+    except V20Error as e:
+        print(e)
+
+
+def cancel_pending_order(accountID, other_args: List[str]):
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        prog="cancelpendingorder",
+        description="Cancel Pending Order",
+    )
+    parser.add_argument(
+        "-i",
+        "--id",
+        dest="orderID",
+        action="store",
+        type=str,
+        required=True,
+    )
+    ns_parser = parse_known_args_and_warn(parser, other_args)
+    if not ns_parser:
+        return
+    try:
+        request = orders.OrderCancel(accountID, ns_parser.orderID)
+        client.request(request)
+    except V20Error as e:
+        print(e)
+
+
+def get_pending_orders(accountID):
+    try:
+        request = orders.OrdersPending(accountID)
+        response = client.request(request)
+        for i in range(len(response["orders"])):
+            try:
+                order_id = response["orders"][i]["id"]
+                instrument = response["orders"][i]["instrument"]
+                price = response["orders"][i]["price"]
+                units = response["orders"][i]["units"]
+                create_time = response["orders"][i]["createTime"]
+                time_in_force = response["orders"][i]["timeInForce"]
+                print(f"Order ID: {order_id}")
+                print(f"Instrument: {instrument}")
+                print(f"Price: {price}")
+                print(f"Units: {units}")
+                print(f"Time created: {create_time}")
+                print(f"Time in force: {time_in_force}")
+                print("-" * 30)
+            except IndexError:
+                break
+    except V20Error as e:
+        print(e)
+
+
+def get_open_positions(accountID):
+    try:
+        request = positions.OpenPositions(accountID)
+        response = client.request(request)
+        for i in range(len(response["positions"])):
+            instrument = response["positions"][i]["instrument"]
+            long_units = response["positions"][i]["long"]["units"]
+            long_pl = response["positions"][i]["long"]["pl"]
+            long_upl = response["positions"][i]["long"]["unrealizedPL"]
+            short_units = response["positions"][i]["short"]["units"]
+            short_pl = response["positions"][i]["short"]["pl"]
+            short_upl = response["positions"][i]["short"]["unrealizedPL"]
+            print(f"Instrument: {instrument}\n")
+            print(f"Long Units: {long_units}")
+            print(f"Total Long P/L: {long_pl}")
+            print(f"Long Unrealized P/L: {long_upl}\n")
+            print(f"Short Units: {short_units}")
+            print(f"Total Short P/L: {short_pl}")
+            print(f"Short Unrealized P/L: {short_upl}")
+            print("-" * 30 + "\n")
+    except V20Error as e:
+        print(e)
+
+
+def get_open_trades(accountID):
+    try:
+        request = trades.OpenTrades(accountID)
```

Comment:
This one also crashed badly for me. 

Even if there are no arguments, it would be good to still use argparse to add a definition to the command so that the user can do:
```
trades -h
```
and see "shows open trades" or something like that